src/sentry/api/endpoints/organization_user_issue.py

```diff
@@ -0,0 +1,231 @@
+        score = self.data.get("score")
+        transaction = self.data.get("transaction", "")
+
+        evidence_data = {
+            "transaction": transaction,
+            "vital": vital,
+            "score": score,
+        }
+
+        evidence_display = [
+            IssueEvidence(
+                name="Transaction",
+                value=transaction,
+                important=False,
+            ),
+            IssueEvidence(
+                name="Web Vital",
+                value=vital.upper(),
+                important=True,
+            ),
+            IssueEvidence(
+                name="Score",
+                value=str(score),
+                important=True,
+            ),
+        ]
+
+        return (evidence_data, evidence_display)
+
+
+ISSUE_TYPE_CHOICES = [
+    WebVitalsGroup.slug,
+]
+
+
+class OrganizationUserIssueRequestSerializer(serializers.Serializer):
+    transaction = serializers.CharField(required=True)
+    projectId = serializers.IntegerField(required=True)
+    issueType = serializers.ChoiceField(required=True, choices=ISSUE_TYPE_CHOICES)
+
+
+class WebVitalsIssueDataSerializer(OrganizationUserIssueRequestSerializer):
+    score = serializers.IntegerField(required=True, min_value=0, max_value=100)
+    vital = serializers.ChoiceField(required=True, choices=["lcp", "fcp", "cls", "fid", "ttfb"])
+
+
+@region_silo_endpoint
+class OrganizationUserIssueEndpoint(OrganizationEndpoint):
+    publish_status = {
+        "POST": ApiPublishStatus.EXPERIMENTAL,
+    }
+    owner = ApiOwner.VISIBILITY
+
+    def get_formatter(self, data: dict) -> BaseUserIssueFormatter:
+        if data.get("issueType") == WebVitalsGroup.slug:
+            return WebVitalsUserIssueFormatter(data)
+        return DefaultUserIssueFormatter(data)
+
+    def get_serializer(self, data: dict) -> serializers.Serializer:
+        if data.get("issueType") == WebVitalsGroup.slug:
+            return WebVitalsIssueDataSerializer(data=data)
+        return OrganizationUserIssueRequestSerializer(data=data)
+
+    def has_feature(self, organization: Organization, request: Request) -> bool:
+        return features.has(
+            "organizations:performance-web-vitals-seer-suggestions",
+            organization,
+            actor=request.user,
+        ) and features.has(
+            "organizations:issue-web-vitals-ingest", organization, actor=request.user
+        )
+
+    def post(self, request: Request, organization: Organization) -> Response:
+        """
+        Create a user defined issue.
+        """
+
+        if not self.has_feature(organization, request):
+            return Response(status=404)
+
+        serializer = self.get_serializer(request.data)
+
+        if not serializer.is_valid():
+            return Response(serializer.errors, status=400)
+
+        validated_data = serializer.validated_data
+        project_id = validated_data.get("projectId")
+        project = Project.objects.get(id=project_id)
```

Comment:
I believe there may be a security issue with this line. https://cheatsheetseries.owasp.org/cheatsheets/Insecure_Direct_Object_Reference_Prevention_Cheat_Sheet.html

(we are not validating this project belongs to the organization that we have access to, so you could create an issue on any arbitrary project).

you should probably just make this endpoint a `ProjectEndpoint` which takes in the org and project id as a url path parameter, so that way the project id and org id are validated for authorization